From 7990b5b92f456431e9c4ccc8e4dac65e56420220 Mon Sep 17 00:00:00 2001
From: Haochen Tong <i@hexchain.org>
Date: Wed, 14 Feb 2024 17:20:44 +0800
Subject: [PATCH] resultmodel: add missing include for QPointer

---
 src/resultmodel.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/resultmodel.cpp b/src/resultmodel.cpp
index 2ae0100..422baa6 100644
--- a/src/resultmodel.cpp
+++ b/src/resultmodel.cpp
@@ -13,6 +13,7 @@
 #include <QDebug>
 #include <QFile>
 #include <QTimer>
+#include <QPointer>
 
 // STL
 #include <functional>
-- 
GitLab

